editor/src/camera/mod.rs

```diff
@@ -83,6 +85,7 @@ pub struct CameraController {
     pitch: f32,
     pub z_offset: f32,
     mouse_control_mode: MouseControlMode,
+    mouse_user_sensitivity: f32,
```

Comment:
It is better to modify `on_mouse_move` method of `CameraController` to accept `settings: &Settings`.

Reply:
Fair enough! I'll make the change